### DISCUSSION TITLE
Add option to use export disparity to export to a JPG image sequence in iw3

### DISCUSSION THREAD

**Discussion by kaelsonofkrypto** [nagadomi/nunif#154] [General]
hi Nagadomi, at the present moment export disparity in iw3 will do a png image sequence on the rgb and depth map, only problem is the png can be so large, that just on 1 hour of footage it can range from 100 to 300gb. I 100 percent am grateful this option exists as this would be best quality, but for those who may not have the space, would you be able to add an export disparity option for the rgb and depth map to jpg since its smaller in file size than png. Please still keep the png option, as i think its still a great option, just asking if we can have a jpg option also?

**Comment by nagadomi** [comment]
This is similar to #145.
The commonly used JPEG format is not a lossless format, and it also does not support 16-bit.

For depth/, it is not recommended as it will cause unrepairable banding artifacts block artifacts.

For rgb/, it will cause color degradation and block artifacts, but that is the same as when encoding video with yuv420, so it is not that bad a choice.
other point to note is that with each edit, unrepairable degradation is accumulated. So in general, using a non-lossless format like JPEG as an intermediate format is not a good choice.

In my opinion, the best way is to simply buy an additional 4T HDD.
If you can accept above problems, it can support JPEG format for rgb/.
